bundles/mapping/mapwfs2/service/UserStyleService.js

```diff
@@ -9,7 +9,7 @@ export class UserStyleService {
             return;
         }
         const layerStyles = this.styles.get(layerId) || [];
```

Comment:
Call this.getUserStylesForLayer(layerId) and change it to `return this.styles.get(layerId) || [];`. This will remove a lot of if statements from the code.